app/javascript/packages/normalize-yaml/index.spec.js

```diff
@@ -51,4 +51,11 @@ describe('normalize', () => {
 
     expect(await normalize(original, { exclude: ['smartPunctuation'] })).to.equal(expected);
   });
+
+  it('allows ignoring specific keys for sorting', async () => {
+    const original = '---\na: 1\nc: 3\nd: 4\nb: 2';
+    const expected = '---\na: 1\nc: 3\nb: 2\nd: 4\n';
```

Comment:
WDYT of multiline strings for readability?
```suggestion
    const original = `---
a: 1
c: 3
d: 4
b: 2`;
    const expected = `---
a: 1
c: 3
b: 2
d: 4
`;
```

Reply:
Sure! The de-indenting always frustrates me with these blocks (want [`String.dedent`](https://github.com/tc39/proposal-string-dedent)!), but I still think it's more readable that way.